src/test/java/com/marizueva/laboratory/hw4/voidpageobject/pages/HomePage.java

```diff
@@ -0,0 +1,169 @@
+    @FindBy(xpath = "//iframe[@id='jdi-frame-site']")
+    WebElement iframeCenter;
+
+    @FindBy(xpath = "//img[@src='images/Logo_Epam_Color.svg']")
+    WebElement epamLogoInIframe;
+
+    @FindBy(xpath = "//div[@class='main-content']/h3[@class='text-center']")
+    WebElement subHeaderJdi;
+
+
+
+    public HomePage(WebDriver driver) {
+        super(driver);
+    }
+
+    @Step("Open site by URL  https://jdi-testing.github.io/jdi-light/index.html")
+    public void openSite() {
+        driver.navigate().to(siteUrl);
+    }
+
+    @Step("login default user: Roman Jdi1234")
+    public void performLogin(String userName, String password) {
+        openLoginDropdownButton.click();
+        loginField.sendKeys(userName);
+        passwordField.sendKeys(password);
+        enterCredentials.click();
+    }
+
+    @Step("Assert User name in the left-top side of screen that user is loggined 'ROMAN IOVLEV'")
+    public String getLoggedUserName() {
+        return loggedUserName.getText();
+    }
+
+    @Step("Assert that there are 4 items on the header section are displayed and\n"
+            + " they have proper texts\n"
+            + "  \"HOME\", \"CONTACT FORM\", \"SERVICE\", \"METALS & COLORS\"")
+    public String getHeaderItem(HeaderItems headerItem) {
+
+        switch (headerItem) {
+            case HOME:
+                return headerItemHome.getText();
+
+            case CONTACT_FORM:
+                return headerItemContact.getText();
+
+            case SERVICE:
+                return headerItemService.getText();
+
+            case METAL_AND_COLORS:
+                return headerItemMetalAndColor.getText();
+
+            default:
+                throw new NotFoundException("There is no such tab" + headerItem);
+
+        }
+    }
+
+    @Step(" Assert that there are 4 images on the Index Page and they are displayed")
+    public boolean isBenefitIconDisplayed(int index) {
+        switch (index) {
+            case 1:
+                return displayedBenefitImage1.isDisplayed();
+            case 2:
+                return displayedBenefitImage2.isDisplayed();
+            case 3:
+                return displayedBenefitImage3.isDisplayed();
+            case 4:
+                return displayedBenefitImage4.isDisplayed();
+            default:
+                throw new NotFoundException("there is no such icon");
+        }
+    }
+
+    @Step("Assert that there are 4 texts on the Index Page under icons\n"
+            + " and they have proper text")
+    public boolean isBenefitTextDisplayed(int index) {
+        switch (index) {
+            case 1:
+                return benefitText1.isDisplayed();
+            case 2:
+                return benefitText2.isDisplayed();
+            case 3:
+                return benefitText3.isDisplayed();
+            case 4:
+                return benefitText4.isDisplayed();
+            default:
+                throw new NotFoundException("there is no such icon");
+        }
+    }
```

Comment:
Why is called assert? I do not see any assertions in this method?

Reply:
fixed